src/benchmarks/micro/corefx/System.Buffers/Base64Tests.cs

```diff
@@ -39,15 +39,15 @@ public void SetupBase64EncodeDestinationTooSmall()
         [Benchmark]
         public OperationStatus Base64EncodeDestinationTooSmall() => Base64.EncodeToUtf8(_decodedBytes, _encodedBytes, out _, out _);
 
-        [GlobalSetup(Target = nameof(Base64EncodeBaseline))]
+        [GlobalSetup(Target = nameof(ConvertToBase64CharArray))]
         public void SetupBase64EncodeBaseline()
```

Comment:
Shouldn't the "Setup" methods be renamed as well?

Reply:
good catch, #done